week1_intro/gym_interface.ipynb

```diff
@@ -151,14 +151,14 @@
    "source": [
     "def policy(obs, t):\n",
     "    # Write the code for your policy here. You can use the observation\n",
-    "    # (a tuple of position and velocity), the current time step, or both,\n",
+    "    # (a tuple of the position and the velocity), the current time step, or both,\n",
     "    # if you want.\n",
     "    position, velocity = obs\n",
     "    \n",
-    "    # This is an example policy. You can try running it, but it will not work.\n",
+    "    # This is an example policy. You can try running it, but it won't work.\n",
     "    # Your goal is to fix that. You don't need anything sophisticated here,\n",
     "    # and you can hard-code any policy that seems to work.\n",
-    "    # Hint: think how you would make a swing go farther and faster.\n",
+    "    # Hint: think how you would make a swing go faster and faster.\n",
```

Comment:
Well, we meant "farther" here. This change doesn't alter the meaning much, though.